src/main/java/org/sonarlint/intellij/ui/CurrentFileConnectedModePanel.java

```diff
@@ -0,0 +1,147 @@
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02
+ */
+package org.sonarlint.intellij.ui;
+
+import com.intellij.ide.IdeTooltipManager;
+import com.intellij.openapi.actionSystem.ActionManager;
+import com.intellij.openapi.application.ApplicationManager;
+import com.intellij.openapi.application.ModalityState;
+import com.intellij.openapi.module.ModuleUtilCore;
+import com.intellij.openapi.project.Project;
+import com.intellij.ui.GuiUtils;
+import com.intellij.ui.TooltipWithClickableLinks;
+import icons.SonarLintIcons;
+import java.awt.CardLayout;
+import java.awt.Cursor;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import org.sonarlint.intellij.common.util.SonarLintUtils;
+import org.sonarlint.intellij.config.global.ServerConnection;
+import org.sonarlint.intellij.core.ModuleBindingManager;
+import org.sonarlint.intellij.core.ProjectBindingManager;
+import org.sonarlint.intellij.exception.InvalidBindingException;
+import org.sonarlint.intellij.util.SonarLintActions;
+
+import static org.apache.commons.lang.StringEscapeUtils.escapeHtml;
+
+public class CurrentFileConnectedModePanel {
+
+  private static final String CONNECTED_MODE_DOCUMENTATION_URL = "https://github.com/SonarSource/sonarlint-intellij/wiki/Bind-to-SonarQube-or-SonarCloud";
+
+  private static final String NOT_CONNECTED = "Not Connected";
+  private static final String CONNECTED = "Connected";
+  private final Project project;
+
+  private JPanel panel;
+  private CardLayout layout;
+  private JLabel connectedCard;
+
+  CurrentFileConnectedModePanel(Project project) {
+    this.project = project;
+    createPanel();
+    switchCards();
+    CurrentFileStatusPanel.subscribeToEventsThatAffectCurrentFile(project, this::switchCards);
+    // TODO Also subscribe to branch management service event
+  }
+
+  private void createPanel() {
+    layout = new CardLayout();
+    panel = new JPanel(layout);
+
+    var notConnectedCard = new JLabel(SonarLintIcons.NOT_CONNECTED);
+    var notConnectedTooltip = new TooltipWithClickableLinks.ForBrowser(notConnectedCard,
+      "<h3>Not Connected</h3>" +
+      "<p>Click to synchronize your project with SonarQube or SonarCloud.</p>" +
```

Comment:
Is it a click on the whole tooltip that opens the configuration panel ? I wonder if it's natural for people to click anywhere on a tooltip, maybe a link would be more explicit ?

Reply:
There _is_ a link in the tooltip. `TooltipWithClickableLinks.ForBrowser` adds a default link handler that opens the link's `href` in a browser window.

Reply:
Are you talking about the _Learn More_ link ? I was talking about this sentence: 
> Click to synchronize your project with SonarQube or SonarCloud 

and the click listener that is set on the card

Reply:
The "card" is the icon, not the tooltip.